src/azure-cli/azure/cli/command_modules/managedservices/custom.py

```diff
@@ -50,11 +44,11 @@ def cli_definition_create(cmd, client,
         Plan=plan,
     )
 
-    return client.create_or_update(
+    poller = client.create_or_update(
```

Comment:
any reason you don't return `poller` here? CLI core has the common logic to detect the case that if it is a poller and will wait to get the result

Reply:
I didnt know that. I have updated all such references to just return the poller.